dv/verilator/sonata_verilator_lint.vlt

```diff
@@ -33,21 +33,17 @@ lint_off -rule UNOPTFLAT -file "*tlul_socket_m1.sv" -match "*Signal unoptimizabl
 // False warning we're using 32-bit initial values for DisableHyperram parameter
 lint_off -rule WIDTHTRUNC -file "*" -match "Operator VAR 'DisableHyperram' expects 1 bits on the Initial value*"
 
+// Unimportant warning where compressed and uncompress instructions are printed in the same place.
+lint_off -rule WIDTHEXPAND -file "*ibex_controller.sv" -match "*Conditional True's VARREF 'instr_compressed_i' generates 16 bits."
+
 // CHERIoT core; reports of circular logic in interactions with the TL-UL components when there are no intervening register stages;
 // these are not combinational loops. UNOPTFLAT warns of slower simulation through incurring multiple simulation delta cycles.
 lint_off -rule UNOPTFLAT -file "*ibex_load_store_unit.sv" -match "*Signal unoptimizable: Circular combinational logic: '*.tbre_req_good'"
 lint_off -rule UNOPTFLAT -file "*ibex_load_store_unit.sv" -match "*Signal unoptimizable: Circular combinational logic: '*.resp_wait'"
 lint_off -rule UNOPTFLAT -file "*ibex_load_store_unit.sv" -match "*Signal unoptimizable: Circular combinational logic: '*.lsu_req_done'"
-lint_off -rule UNOPTFLAT -file "*ibex_id_stage.sv" -match "*Signal unoptimizable: Circular combinational logic: '*.lsu_req'"
-lint_off -rule UNOPTFLAT -file "*ibex_controller.sv" -match "*Signal unoptimizable: Circular combinational logic: '*.load_err_d'"
-lint_off -rule UNOPTFLAT -file "*ibex_controller.sv" -match "*Signal unoptimizable: Circular combinational logic: '*.store_err_d'"
-lint_off -rule UNOPTFLAT -file "*ibex_core.sv" -match "*Signal unoptimizable: Circular combinational logic: '*.rv32_lsu_req'"
-lint_off -rule UNOPTFLAT -file "*ibex_core.sv" -match "*Signal unoptimizable: Circular combinational logic: '*.lsu_is_cap'"
-lint_off -rule UNOPTFLAT -file "*ibex_core.sv" -match "*Signal unoptimizable: Circular combinational logic: '*.lsu_req'"
-lint_off -rule UNOPTFLAT -file "*ibex_core.sv" -match "*Signal unoptimizable: Circular combinational logic: '*.lsu_we'"
```

Comment:
This is good news, looks like there's been a bunch of work on this logic to clean it up given these warnings have gone away :)

Reply:
Yes, I was very happy to remove some of the UNOPTFLAT waivers :)